random-csv-data-set/src/main/java/com/blazemeter/jmeter/RandomCSVDataSetConfigGui.java

```diff
@@ -0,0 +1,198 @@
+package com.blazemeter.jmeter;
+
+import kg.apc.jmeter.JMeterPluginsUtils;
+import kg.apc.jmeter.gui.BrowseAction;
+import kg.apc.jmeter.gui.GuiBuilderHelper;
+import org.apache.jmeter.config.gui.AbstractConfigGui;
+import org.apache.jmeter.testelement.TestElement;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class RandomCSVDataSetConfigGui extends AbstractConfigGui {
+
+    public static final String WIKIPAGE = "RandomCSVDataSetConfig";
```

Comment:
Shouldn't it contain full URL of wiki page?

Reply:
We have some problems here, because kg.apc.jmeter.JMeterPluginsUtils#addHelpLinkToPanel always add this prefix: WIKI_BASE = "http://jmeter-plugins.org/wiki/"

Reply:
Not always: https://github.com/undera/jmeter-plugins/blob/ec7f8cea9d86e6a21db8ac9b1d3100a6c4936215/infra/common/src/main/java/kg/apc/jmeter/JMeterPluginsUtils.java#L175

Latest commons lib allows having `http://` addresses

Reply:
This version doesn't released. 
Version 0.5 was released in May. This method added in June. Let's release?

Reply:
http://search.maven.org/#artifactdetails%7Ckg.apc%7Cjmeter-plugins-cmn-jmeter%7C0.5%7Cjar
Released several months ago...